src/main/java/org/tikv/common/TiSession.java

```diff
@@ -196,7 +196,7 @@ public Catalog getCatalog() {
     return res;
   }
 
-  public synchronized RegionManager getRegionManager() {
+  public RegionManager getRegionManager() {
```

Comment:
any reason to remove `synchronized` ?

Reply:
Because it is safe to read the pointer concurrently. Any change for this pointer will be in synchronized.

Reply:
Just like the method `getPDClient`